x-pack/plugins/apm/public/components/app/service_overview/service_overview_dependencies_table/index.tsx

```diff
@@ -134,12 +135,32 @@ export function ServiceOverviewDependenciesTable({
       dependencies={dependencies}
       fixedHeight={fixedHeight}
       isSingleColumn={isSingleColumn}
-      title={i18n.translate(
-        'xpack.apm.serviceOverview.dependenciesTableTitle',
-        {
-          defaultMessage: 'Dependencies',
-        }
-      )}
+      title={
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.serviceOverview.dependenciesTableTitleTip',
+            {
+              defaultMessage:
+                'Uninstrumented downstream services or external connections derived from the exit spans of instrumented services.',
+            }
+          )}
+        >
+          <>
+            {i18n.translate(
+              'xpack.apm.serviceOverview.dependenciesTableTitle',
+              {
+                defaultMessage: 'Dependencies',
+              }
+            )}{' '}
```

Comment:
```suggestion
            )}
```

extra space character may not be necessary here. If it breaks without it, instead use the space entity:

```suggestion
            )}&nbsp;
```

Reply:
It doesn't break without it, but it does feel a little tight to me. WDYT? FTR has no space, and Impact has a space:

<img width="829" alt="Screen Shot 2022-03-18 at 10 01 04 AM" src="https://user-images.githubusercontent.com/5618806/159049026-1dd08b97-f28a-4202-8fad-c54a3a8396a4.png">